Fixed issues with captions of certain tutorials (#533)
NCAS-CMS/cf-python: docs/source/tutorial.rst

date-time objects.

.. code-block:: python

>    :caption: *Create subspaces in different ways based on the time dimension by selecting a particular date and using date-time and its queries.*

Reviewer: ```suggestion
   :caption: *Create subspaces in different ways based on the time dimension by selecting a particular date and using date-time and date-time queries.*
```

  are removed to create the returned subspace:

  .. code-block:: python

>      :caption: *Create subspace by compressing the domain spanning the 1st, 2nd, 4th and 6th elements of the 'X' axis, with the other domain axes remaining unchanged.*

Reviewer: ```suggestion
     :caption: *Create a subspace by compressing the domain spanning the 2nd, 3rd, 5th and 7th elements of the 'X' axis, with the other domain axes remaining unchanged.*
```

  unselected locations within the envelope.

  .. code-block:: python

>      :caption: *Create subspace by selecting the 1st, 2nd, 4th and 6th elements of the 'X' axis by creating an envelope of these elements and mising data within the envelope wherever needed.*

Reviewer: ```suggestion
     :caption: *Create a subspace by selecting the 2nd, 3rd, 5th and 7th elements of the 'X' axis by creating an envelope of these elements with missing data within the envelope wherever needed.*
```

  locations.

  .. code-block:: python

>      :caption: *Create subspace by selecting the 1st, 2nd, 4th and 6th elements of the 'X' axis with domain encompassing that of the original field contruct and mising data within the domain wherever needed.*

Reviewer: ```suggestion
     :caption: *Create a subspace by selecting the 2nd, 3rd, 5th and 7th elements of the 'X' axis with domain encompassing that of the original field construct with missing data within the domain wherever needed.*
```

constructs

.. code-block:: python

>    :caption: *Create subspace whose domain spans latitudes within the range of 51 to 53 degrees north, with the other domain axes remaining unchanged.*

Reviewer: ```suggestion
   :caption: *Create a subspace whose domain spans latitudes within the range of 51 to 53 degrees north, with the other domain axes remaining unchanged.*
```

the context of that object.

.. code-block:: python

>    :caption: *An example evaluating a strictly less than 3 query operation.*

Reviewer: ```suggestion
   :caption: *An example evaluating a strictly less than 3 query condition.*
```

object.

.. code-block:: python

>    :caption: *An example evaluating compound query operation involving a greater than or equal to 3 and a strictly less than 5 condition.*

Reviewer: ```suggestion
   :caption: *An example evaluating a compound query operation involving a greater than or equal to 3 condition and a strictly less than 5 condition.*
```
